examples/multi_platform/MixedLibWithHeaderMap/MixedAnswer.m

```diff
@@ -0,0 +1,11 @@
+#import <MixedAnswer/MixedAnswer.h>
+
+#import "examples/multi_platform/MixedLibWithHeaderMap/MixedAnswer-Swift.h"
```

Comment:
Want to use header maps in the entire example?

Reply:
We cant yet until we have #2371, it requires knowing the swift_library internal target and passing that to `deps` of `header_map`. It needs to be done by `experimental_mixed_language_library`

Reply:
Wonderful, sounds good to me then!